packages/flame_riverpod/pubspec.yaml

```
@@ -0,0 +1,19 @@
+name: flame_riverpod
+description: Model classes and an example project for using Riverpod in conjunction with Flame.
```

```suggestion
description: Helpers for using Riverpod in conjunction with Flame.
```
Taken from the barrel file, since the example project has it's own description field it doesn't need to be mentioned here.